docs/content/installation/install_driver_operatorhub.md

```diff
@@ -2,6 +2,10 @@
 
 When using OperatorHub.io, the operator for IBM® block storage CSI driver can be installed directly from the OperatorHub.io website. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
 
-To install the CSI driver from OperatorHub.io, go to https://operatorhub.io/operator/ibm-block-csi-operator-community and follow the installation instructions, once clicking the **Install** button.
+1. Install the CSI operator from OperatorHub.io, go to https://operatorhub.io/operator/ibm-block-csi-operator-community and follow the installation instructions, once clicking the **Install** button.
 
-**Note:** To ensure that the operator installs the driver, be sure to apply the YAML file that is located as part of the ibm-block-csi-operator-community page mentioned above.
+2. Apply the IBMBlockCSI custom resource definition yaml provided.
+
+    **Note:** To ensure that the operator installs the driver, be sure to apply the YAML file that is located as part of the ibm-block-csi-operator-community page mentioned above.
+
+3. (Optional) Apply the HostDefiner custom resource definition yaml provided. For more information, see [Configuring the host definer](../configuration/configuring_hostdefiner.md).
```

Comment:
@matancarmeli7 shouldn't the order of steps be reversed?

Reply:
yes you're right, we had a comment to align it no?

Reply:
>yes you're right, we had a comment to align it no?
you mean when you [falsely claimed the installation steps should be in the same order](https://github.com/IBM/ibm-block-csi-driver/pull/517#discussion_r922839631) as the uninstallation steps?

please sync with @rivkap regarding the steps order for all un/installation methods